### PR TITLE
doc: "docker getting started" mentions `flash` limitations

### PR DESCRIPTION
**Pull request by sotte**
## Description

Little doc improvement for the docker getting started guide as result of a discussion on discord.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

none 

## Checklist

- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/getting_started_docker.md [hunk 1]
```diff
@@ -32,6 +34,8 @@ util/docker_build.sh keyboard:keymap:target
 # For example: util/docker_build.sh planck/rev6:default:flash
 ```
 
+Note that mass storage bootloader are not supported by the `flash` target. In this case you have to manually copy over the firmware file to the keyboard.
```
Comment (noroadsleft) [inline docs/getting_started_docker.md:37]: I think this reads better: 

```suggestion
Note that mass storage bootloaders are not supported by the `flash` target. In this case you will have to manually copy the firmware file to the keyboard.
```

**Comment by sotte** [comment]
Thanks for the review @noroadsleft.

**Review by fauxpark — APPROVED** [review]

**Comment by sotte** [comment]
Is there anything else I can do to help out? From my PoV this PR is ready to be merged.

**Review by waffle87 — APPROVED** [review]

**Comment by sotte** [comment]
Thanks for merging this!

**Pull request merged**